examples/hp_search_benchmarks/darts_cifar10_pytorch/model_def.py

```diff
@@ -149,7 +157,9 @@ def train_batch(
     ) -> Dict[str, torch.Tensor]:
         input, target = batch
         model.drop_path_prob = (
-            self.hparams["drop_path_prob"] * (self.scheduler.last_epoch) / self.hparams["train_epochs"]
```

Comment:
I actually kind of like how this was in one line before (seems more readable to me)

Reply:
This was outputted by black formatting.  I think okay to keep.